### DISCUSSION TITLE
How does pricing work?

### DISCUSSION THREAD

**Discussion by IuriiZh** [deepgram/community#266] [General help]
I made next request
REQUEST ID d796cc11-fd98-4edf-b3dd-0cd74cb7c7be
STATUS 200 OK
PERFORMED AT 07/12/2023 17:49:29
Audio duration 02:32:10
Cost $1.31892
Model 85b5dbfd-bc1d-46b5-99d8-070d379a7f97
Method Synchronous
Tier nova

Detect Topics True
NER True
Numerals True
Paragraphs True
Punctuate True
Smart Format True
Summarize True

Duration 2 hours 32 this is 152 min
$1.31892 / 152 = $0.008677 per minute

I didn't find any similar price per minute.

Please help me find out how pricing is work...

**Comment by jjmaldonis** [comment]
Hey @IuriiZh, the reason for this pricing is because Nova is $0.0044/minute and Summarize (which is an Audio Intelligence feature) is an additional $0.0043/minute. That brings the price up to $0.0087/minute, which is what you are seeing.

I pulled these numbers from the "Pay as You Go" column on this page: https://deepgram.com/pricing

**Comment by IuriiZh** [comment]
Thank yo so much
